dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -281,6 +379,32 @@ def test_options_stop_words(self):
         self.assertDictEqual(expected_word_count, text_profile.word_count)
         self.assertDictEqual(expected_vocab, text_profile.vocab_count)
 
+    def test_options_vocab_update(self):
+        # change vocab.is_enabled, other options remain the same as default values
+
```

Comment:
It seems like this comment is meant for this block, not the entire function, did you mean to add a new line here?